TreeDB/db/storage_maintenance_plan.go

```
@@ -0,0 +1,41 @@
+}
+
+func validStorageMaintenancePlan(plan StorageMaintenancePlan) bool {
+	if plan == nil {
+		return false
+	}
+	if token, ok := plan.(storagemaintenance.Plan); ok {
+		return storagemaintenance.IsColumnAssetRewritePlan(token)
+	}
+	value := reflect.ValueOf(plan)
+	switch value.Kind() {
+	case reflect.Chan, reflect.Func, reflect.Interface, reflect.Map, reflect.Ptr, reflect.Slice:
```

golangci-lint (2.12.2)

[error] inline: Constant reflect.Ptr should be inlined

(govet)